### DISCUSSION TITLE
How can change color of text of note_floating(position, text) function?

### DISCUSSION THREAD

**Discussion by Mehrshad-Ghorbanzadeh** [jimy-byerley/pymadcad#14] [General]
Hi, Can you help me, please? I can not change the color of the text in a note_floating function. 
and also how can change the color of a segment?

**Comment by jimy-byerley** [comment]
Hello @MehrshadGh 
At the moment, no function `note_*` has a color option. Not that it would be hard to add, but my initial intention was to provide a very simple way to display annotations, with minimalistic settings. So it's using the annotation color set in the settings file. 

If you want to change all annotation colors, you should change your settings
[madcad.settings](https://github.com/jimy-byerley/pymadcad/blob/2e7195aa064d2b99abcaf61067ce8d4f5d0ab1fe/madcad/settings.py#L16) defines global variables with the settings from `~/.config/madcad/pymadcad.yaml`. You can change either the config file, or the global variable in your script.

If you only need a floating text with a color different from annotations, you can directly use [`madcad.text.Text`](https://github.com/jimy-byerley/pymadcad/blob/2e7195aa064d2b99abcaf61067ce8d4f5d0ab1fe/madcad/text.py#L39) which is a bit more customizable (not so much though)

The same applies for `Segment`, the color is only set by the global settings, or you need to deal with a lower level display

What is your need ?

**Reply by Mehrshad-Ghorbanzadeh** [reply]
Thank you so much. now, I understood and use Text to do color change and size. 
The other question was about the color of a segment. Can I Change it?

**Reply by jimy-byerley** [reply]
You can only change the color of a segment, by changing the color of all segments in the settings.
If you want to tune the color for a specific one, you must go to the lower level:  the display object itself (like `Text` but for Webs), called [`WebDisplay`](https://github.com/jimy-byerley/pymadcad/blob/2e7195aa064d2b99abcaf61067ce8d4f5d0ab1fe/madcad/displays.py#L359)

But If your are looking for a way to create schemes with different colors, font size, animations etc, I would advice to use [`Scheme`](https://pymadcad.readthedocs.io/en/latest/reference/scheme.html#madcad.scheme.Scheme) instead. That last object is meant for it.
